docs/feature/storage/indexing-and-storage.md

```diff
@@ -0,0 +1,320 @@
+(indexing-and-storage)=
+
+# Indexing and storage in CrateDB
+
+:::{article-info}
+---
+avatar: https://avatars.githubusercontent.com/u/7926726?v=4
+avatar-link: https://github.com/marijaselakovic
+avatar-outline: muted
+author: Marija Selakovic
+date: November 12, 2021
+read-time: 8 min read
+class-container: sd-p-2 sd-outline-muted sd-rounded-1
+---
+:::
+
+## Introduction
+
+This article explores the internal workings of the storage layer in CrateDB.
+The storage layer ensures that data is stored in a safe and accurate
+way and returned completely and efficiently.
+CrateDB's storage layer is based on Lucene indexes.
+
+## What's inside
+
+Lucene offers scalable and high-performance indexing, which enables efficient
+search and aggregations over documents and rapid updates to the existing
+documents. We will look at the three main Lucene structures that are used
+within CrateDB: Inverted indexes for text values, BKD trees for numeric
+values, and doc values.
+
+:Inverted index: You will learn how inverted indexes are implemented in Lucene
```

Comment:
I don't like here the `You will learn` phrasing, as this is more appropriate for a live presentation agenda slide.
Although this comes from Marija's context, I'd suggest to rephrase it:
```
Explain how inverted indexes are implemented in Lucene and how CrateDB uses them to index text values and enable fast text searches.
```

Reply:
Thanks, I agree. Even if it's an article pulled in from another stage, we shouldn't stop curating it going forward.

I've used your suggestion 1:1, but after some little back and forth, just started using "understand" to begin the sentence, picked up from the definition item of the `:BKD tree:` term, as you didn't have any complaints about that, see a13dfb24f87db.

Well, it's a repetition of words, but on the other hand, it's symmetric right now. ;]